<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalenderPage</class>
 <widget class="QWidget" name="CalenderPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_cp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>紀錄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>郵件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_up">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#evilButton {
    background-color: rgb(200,200,200);
}</string>
   </property>
   <property name="text">
    <string>使用者</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>郵件號碼:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
